.meshrc.yaml

```diff
@@ -13,6 +13,20 @@ sources:
         source: ./sources/czid-schema.graphql
         operationHeaders:
           Cookie: "{context.headers['cookie']}"
+  - name: NextGenEntities
+    handler:
+      graphql:
+        endpoint: ${NEXTGEN_API_URL}/graphql
+        source: ./sources/nextgen-entities-schema.graphql
+        operationHeaders:
+          Cookie: "{context.headers['cookie']}"
```

Comment:
This is already configured above, as `CZIDENTITIES`.  i don't know if they are exactly the same, but we should figure out which one is most up to date.  I think @rzlim08 added the one above.

Reply:
Thanks for the call out. I updated the details here in the two sources from Ryan's and removed that one. Will update the schemas as well